tests/foreman/cli/test_contentviews.py

```diff
@@ -1055,16 +980,22 @@ def test_cv_promote_composite(self):
             self.fail(err)
 
         # Associate repo to CV
-        result = ContentView.add_repository({u'id': new_cv['id'],
-                                             u'repository-id': new_repo['id']})
-        self.assertEqual(result.return_code, 0,
-                         "Repo was not associated to selected CV")
+        result = ContentView.add_repository({
+            u'id': new_cv['id'],
+            u'repository-id': new_repo['id'],
+        })
+        self.assertEqual(
+            result.return_code, 0,
+            "Repo was not associated to selected CV"
+        )
         self.assertEqual(len(result.stderr), 0, "No error was expected")
 
         # Publish a new version of CV
         result = ContentView.publish({u'id': new_cv['id']})
-        self.assertEqual(result.return_code, 0,
-                         "Publishing a new version of CV was not successful")
+        self.assertEqual(
+            result.return_code, 0,
+            "Publishing a new version of CV was not successful"
```

Comment:
You have removed the third parameter, the error message that gets displayed to the user, in other parts of the code, so perhaps you should remove this one here too?